src/tests/compare/test_compare.py

```diff
@@ -43,7 +43,8 @@ def test_compare_fails_on_missing_dependencies(self, storage_mock, model_mock):
         }
 
         result = compare(message)
-        self.assertEqual(result, None)
+        print(result)
```

Comment:
I guess this print statement was used when writing the tests, can it be removed?